Tasks/GooglePlayReleaseBundle/package.json

```diff
@@ -11,10 +11,10 @@
   "dependencies": {
     "@types/node": "^10.17.0",
     "@types/mocha": "^5.2.7",
-    "azure-pipelines-task-lib": "^3.0.6-preview.1",
+    "azure-pipelines-task-lib": "^3.1.0",
     "glob": "~7.1.4",
-    "google-auth-library": "5.2.1",
-    "googleapis": "~43.0.0"
+    "googleapis": "^67.1.0",
+    "google-auth-library": "^7.0.2"
```

Comment:
Do we need to use it like ^ instead of sticking to specific version?

Reply:
I wanted to make a system for indicating the versions of these libraries for this task in the same format as for other tasks in the extension. In other tasks in the extension libs versions are specified using ^. Generally, I think it quite safe because based on the release notes of these libs any breaking changes are introducing with bumping of major versions, minor and patch version contains some kind of bug fix and minor features.